package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "nodemon",
     "start": "node ./dist/server/app.js",
     "build": "tsc -p tsconfig.prod.json",
-    "test": "NODE_ENV=test && env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register 'test/**/*.ts' --timeout 10000 --exit",
+    "test": "SET NODE_ENV=test && env TS_NODE_COMPILER_OPTIONS='{\"\"\"module\"\"\":\"\"\"commonjs\"\"\"}' mocha -r ts-node/register 'test/**/*.ts' --timeout 10000 --exit",
```

Comment:
Could you revert this to the original script, we can handle the local setup for windows users in a separate PR by creating a different script and adding some documentation in the Readme.
If you are interested you can create an issue with ``hard`` label for the same once this is merged and get it assigned.